### PR TITLE
Desktop Voice Assistant

### LINKED ISSUE #709: Desktop Voice Assistant
# Aim

To be able to perform basic functions on our desktop using our voice

# Details

- What the features will your script have
Will be able to perform basic functions like:
1. Sending email
2. Opening google
3. opening some app
or any other

### PR DESCRIPTION
**Pull request by sayantani11**
# Description

This is a desktop voice assistant that will help you automate a lot of task on your device. Before you start the assistant its advised to read the readme file and the other text files to make the required changes.

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation Update

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [ ] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

**Comment by sayantani11** [comment]
I don't know why the lint code fails because I did run the code in my device.
Can the mentors please suggest?

**Comment by sayantani11** [comment]
I don't know it works fine in my device, can someone tell the underlying problem if possible

**Comment by kaustubhgupta** [comment]
@sayantani11  

image

**Comment by sayantani11** [comment]
@kaustubhgupta I ran it in jupyter lab, it was working fine.....Lemme see

**Comment by sayantani11** [comment]
@kaustubhgupta have a look, if any changes are required.....do inform

File: Desktop-Voice-assistant/READme.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# DESKTOP VOICE ASSISTANT #
```
Comment (kaustubhgupta) [inline Desktop-Voice-assistant/READme.md:1]: Remove (#) at the end from every heading of this readme

File: Desktop-Voice-assistant/READme.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+# LEVEL -2 #
```
Comment (kaustubhgupta) [inline Desktop-Voice-assistant/READme.md:3]: remove this line

File: Desktop-Voice-assistant/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#Following modules are being used here:
```
Comment (kaustubhgupta) [inline Desktop-Voice-assistant/requirements.txt:1]: remove this line

File: Desktop-Voice-assistant/important.txt [hunk 4]
```diff
@@ -0,0 +1,16 @@
+Here for making the sent Email function work, there are certain changes that should be made
```
Comment (kaustubhgupta) [inline Desktop-Voice-assistant/important.txt:1]: shift this content to readme file and remove this file

File: Desktop-Voice-assistant/other.txt [hunk 5]
```diff
@@ -0,0 +1,8 @@
+You can use it to open apps on your device like word, one note, notepad, VS code etc
```
Comment (kaustubhgupta) [inline Desktop-Voice-assistant/other.txt:1]: shift this content to readme file and remove this file

File: pywhatkit_dbs.txt [hunk 6]
```diff
@@ -0,0 +1 @@
+--------------------
```
Comment (kaustubhgupta) [inline pywhatkit_dbs.txt:1]: remove this file

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

File: pywhatkit_dbs.txt (hunk 6 above)
Comment (sayantani11) [reply pywhatkit_dbs.txt:1]: I don't even know where this file is....I will try removing it

File: Desktop-Voice-assistant/READme.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
+# DESKTOP VOICE ASSISTANT
+
+# LEVEL -2
```
Comment (kaustubhgupta) [inline Desktop-Voice-assistant/READme.md:3]: remove this line

File: Desktop-Voice-assistant/READme.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+    for no privacy breach we can use the following steps:
+    1. Go to goole and search "App passwords google"
+    2. Open "App passwords-Sign in Google"
+    3. Sign in the with the account you want to use for sending emails using this
+    4.Under "select the app and device you want generate password for"
+         i> Select App= Mail   Select Device= Windows 10
+         ii>Click on generate
+    5. Copy the app password and use it here.
+
+# Check requirements.txt before using the assistant
```
Comment (kaustubhgupta) [inline Desktop-Voice-assistant/READme.md:61]: remove this line and blank lines ahead

File: Desktop-Voice-assistant/READme.md [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+
+
+# Development Status #
```
Comment (kaustubhgupta) [inline Desktop-Voice-assistant/READme.md:64]:
```suggestion
# Development Status 
```

File: Desktop-Voice-assistant/READme.md [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+
+The amount of things that can be done with this AI assistant is endless. This is just a basic structure which is at present complete. It might be using hotwords in future that will trigger the assistant, , might be used in sending the system to sleep mode, set alarm, send text messages to contacts on your phone and lot more.
+
+
+# Developed by Sayantani Saha #
```
Comment (kaustubhgupta) [inline Desktop-Voice-assistant/READme.md:69]:
```suggestion
# Developed by Sayantani Saha 
```

File: Desktop-Voice-assistant/voice_assistant.py [hunk 11]
```diff
@@ -0,0 +1,138 @@
...
+import sys
+
+print("Your Assistant is starting......")
+
+engine = pyttsx3.init('sapi5')
+voices= engine.getProperty('voices')
+engine.setProperty('voice', voices[1].id)
+engine.setProperty('rate', 190)
+
+Master="Your Name"  # when you start using the assistant enter your name here
```
Comment (kaustubhgupta) [inline Desktop-Voice-assistant/voice_assistant.py:20]: take this as input from user

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Review by kaustubhgupta — APPROVED** [review]

**Comment by kaustubhgupta** [comment]
@Kushal997-das

**Comment by kaustubhgupta** [comment]
@Kushal997-das @santushtisharma10 @antrikshmisri

**Review by antrikshmisri — APPROVED** [review]

**Review by santushtisharma10 — APPROVED** [review]

**Review by Kushal997-das — APPROVED** [review]
LGTM 💯

**Review by avinashkranjan — APPROVED** [review]
LGTM..💯

**Pull request merged**